### ISSUE TITLE
1.1.0 windows portable zip not available?

### ISSUE THREAD

**Issue by Juesto** [kritzikratzi/Oscilloscope#79]
Looks maybe it was forgotten to be included on the release since only the osx version and the installer file is there for download

**Comment by kritzikratzi** [comment]
you were using the portable version? i was under the impression that installers are preferred by almost everyone (i got a lot of confused "i don't need to install it?" over the years)

**Comment by Juesto** [comment]
yes i'm those people who prefer portable or extracting , it's good to offer both instances for different significant audiences

**Comment by IsaacShoebottom** [comment]
I am writing a [scoop manifest](https://github.com/IsaacShoebottom/scoop-bucket/blob/master/bucket/oscilloscope.json) for this app and it would be helpful to have a portable install. Extracting the MSI results in a non functional app for some reason. 1.0.8 remains functional in portable mode

**Comment by kritzikratzi** [comment]
i have no date, but i can easily bring back the portable "installer" with the next version.

**Comment by IsaacShoebottom** [comment]
It's not required, although it would be nice to have the option, in the future. Extracting the msi works. I marked it as outdated because it was not the extraction process that results in a non functional app. It's launching the app from a symlinked directory, which is the way that scoop points to the most recent version of an app. The app works fine when launched through the non-symlinked directory. If I don't forget I could make an issue, but this is a pretty niche use case tbh.

The easiest way to see how it breaks is probably to boot up windows sandbox (or a standard windows vm) and install scoop and my repo, and launch the app
